client/src/balrogclient/api.py

```diff
@@ -16,8 +16,8 @@
 
 def _json_log_data(data):
     log = json.dumps(data)
-    if len(log) > 100:
```

Comment:
I guess maybe this should be a constant?

Reply:
Good idea.